articles/libraries/lock-android/passwordless.md

```diff
@@ -6,7 +6,9 @@ description: Guide on implementing Passwordless authentication with Lock for And
 # Lock Passwordless
 
 ::: warning
-This feature is disabled by default for new tenants as of 8 June 2017. If you would like this feature enabled, please contact support to discuss your use case and prevent the possibility of introducing security vulnerabilities. Please see [Client Grant Types](/clients/client-grant-types) for more information.
+Passwordless on native platforms is disabled by default for new tenants as of 8 June 2017. If you would like this feature enabled, please contact support to discuss your use case and prevent the possibility of introducing security vulnerabilities. Please see [Client Grant Types](/clients/client-grant-types) for more information.
```

Comment:
Also the `check into` does not seem right, it's like checking into a hotel or sth.